src/pretix/base/forms/questions.py

```diff
@@ -88,21 +91,21 @@ def __init__(self, *args, **kwargs):
                 field = forms.DecimalField(
                     label=q.question, required=q.required,
                     help_text=q.help_text,
-                    initial=initial.answer if initial else None,
+                    initial=initial.answer if initial else default if default else None,
```

Comment:
Nested ``if`` statements of this kind are very hard to read, could you add in some ``()`` to make the order of execution clear?